examples/build.gradle.kts

```diff
@@ -31,9 +31,9 @@ allprojects {
     }
 
     repositories {
+        mavenLocal()
```

Comment:
I think we should use 
`maven(url = uri("https://packages.jetbrains.team/maven/p/reflekt/reflekt"))`
in this examle or add some description in README how publish to mavenLocal and use Reflekt